lib/taxonomy/world_taxonomy.rb

```diff
@@ -6,11 +6,17 @@ def initialize(adapter: RedisCacheAdapter.new, tree_builder_class: Tree)
     end
 
     def all_world_taxons
-      @_world_taxon_list ||= world_taxon_branches
+      @_all_world_taxons ||= sort_world_taxons
     end
 
   private
 
+    def sort_world_taxons
```

Comment:
I'd expect `sort_world_taxons` to take some taxons, and sort them. Maybe `sorted_world_taxons` would be a better name for the way this is being used?